### DISCUSSION TITLE
WIP - a different way to control

### DISCUSSION THREAD

**Discussion by jontelang** [rxhanson/Rectangle#882] [Show and tell]
I have used a custom hacked implementation for a while that let me change the drag areas to allow for some different sizes (eg replacing corners with 75%/25%) but it was a pain to drag the windows so far (fairly large screen) so I made this:

It displays a couple of options when you start to drag, so that you don't need to move your mouse too much.

https://user-images.githubusercontent.com/899947/185427718-df0173c2-75f1-4edc-a410-c0aa5c5afec8.mp4

Thoughts?

**Comment by rxhanson** [comment]
This is great! I was actually planning on building this into Rectangle Pro in an upcoming release (with some additional bells & whistles like configuration of the window actions and the location of the popup). If you get to a point where you want to roll it into Rectangle, I see no reason to turn it down. I will still be adding my version of it into Rectangle Pro regardless at some point, but it's going to take me some time.

**Reply by jontelang** [reply]
That's actually great to hear. I am procrastinating more than I should already, so I don't think I should spend more time on this. 

I have subscribed to the 'Pro' repo so hopefully I will see it when it releases someday, definitely will purchase if it works like this.
